src/t2iapi/alert/alert_requests.proto

```diff
@@ -52,12 +52,4 @@ message AlertConditionEscalationRequest {
   AlertConditionEscalationProcess escalation_process = 2;
 }
 
-/*
-This message is used as a request to trigger a MDIB change, which causes the AlertCondition corresponding to the given
-handle to change its Presence to the given value.
- */
-message SetAlertConditionAndReturnCauseRequest {
```

Comment:
should we add the deleted manipulation to changelog?

Reply:
Yes, why not?

Reply:
@yafbu please add the changes to changelog